Gemfile

```diff
@@ -4,7 +4,7 @@ source 'https://rubygems.org'
 gem 'addressable'
 gem 'onlyoffice_documentserver_conversion_helper'
 gem 'onlyoffice_logger_helper'
-gem 'onlyoffice_s3_wrapper'
+gem 'onlyoffice_s3_wrapper', '>=0.6.0'
```

Comment:
Add a space after >= so it looks like on line 98 `'>= 0.6.0'`
Just a minor code style problem